contentcuration/contentcuration/view/node_views.py

```diff
@@ -341,6 +341,7 @@ def _duplicate_node_bulk_recursive(node, sort_order, parent, channel_id, to_crea
     new_node.source_channel_id = node.get_channel().id if node.get_channel() else None
     new_node.node_id = uuid.uuid4().hex
     new_node.source_node_id = node.node_id
+    new_node.freeze_authoring_data = not Channel.objects.filter(pk=node.original_channel_id, editors=user).exists()
```

Comment:
See if user can edit channel to determine whether or not to freeze authoring data

Reply:
Maybe exclude if current user is a superuser/admin, too?

Reply:
Is there a use case for this? Also, do we need to worry about an admin importing from a public channel into a channel where no one has edit access to the public channel's content?

Reply:
Good point. We can always go in and toggle it after if needed, too.